style.css

```diff
@@ -32,36 +105,43 @@ body {
     width: 90%;
     max-width: 100vh;
     margin: 1.6em auto;
-
-    display: grid;
-    gap: 1.6em;
-    grid-template-columns: repeat(5, 1fr);
+    border: 3px black solid;
+    display: flex;
+    flex-direction: row;
+    padding: 10px;
+    
 }  
 
-/* PARA HEADER, CONTENT, SIDEBAR, ETC. QUE ESTA DENTRO DE CONTAINER  */
+.relatedPostParent {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    width: 90%;
+    max-width: 100vh;
+    margin: 1em auto;
+    border: 3px black solid;
+    padding: 10px;
+    background: #FFC300;
+}
+
+.relatedPostChild {
+    max-width: fit-content;
+    
 
-.container > .header,
-.container > .content,
-.container > .sidebar,
-.container > .footer,
-.container > div {
-    background: #FFC333;
-    padding: 20px;
-    border: 1px solid #000000;
 }
 
-.container > .header,
-.container > .footer {
-    grid-column: 1/6;
-    background: #1e272e;
-    color: #fff;
+.footer{
+    padding: 20px 0;
+    text-align: center;
+    font-size: 1em;
 }
 
-.container > .content{
-    grid-column: span 3;
-} 
 
-.container > .sidebar{
-    grid-column: span 2;
-} 
+
+
+
+
+
+
+
```

Comment:
Use prettier so there's no excess of empty spaces. Also, remove comments when pushing code for a PR. The only comments that are permitted are `TODO: Need fixing yadayadyadyad`